cfn-resources/stream-instance/mongodb-atlas-streaminstance.json

```diff
@@ -0,0 +1,245 @@
+        },
+        "Type": {
+          "type": "string",
+          "description": "Type of the DB role. Can be either BuiltIn or Custom.",
+          "enum": [
+            "BUILT_IN",
+            "CUSTOM"
+          ]
+        }
+      },
+      "additionalProperties": false
+    },
+    "StreamsKafkaSecurity": {
+      "type": "object",
+      "description": "Properties for the secure transport connection to Kafka. For SSL, this can include the trusted certificate to use.",
+      "properties": {
+        "BrokerPublicCertificate": {
+          "type": "string",
+          "description": "A trusted, public x509 certificate for connecting to Kafka over SSL."
+        },
+        "Protocol": {
+          "type": "string",
+          "description": "Describes the transport type. Can be either PLAINTEXT or SSL."
+        }
+      },
+      "additionalProperties": false
+    },
+    "StreamsConnection": {
+      "type": "object",
+      "description": "Settings that define a connection to an external data store.",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "Human-readable label that identifies the stream connection."
+        },
+        "Type": {
+          "type": "string",
+          "description": "Type of the connection. Can be either Cluster or Kafka.",
+          "enum": [
+            "Kafka",
+            "Cluster",
+            "Sample"
+          ]
+        },
+        "Authentication": {
+          "$ref": "#/definitions/StreamsKafkaAuthentication"
+        },
+        "BootstrapServers": {
+          "type": "string",
+          "description": "Comma separated list of server addresses."
+        },
+        "Security": {
+          "$ref": "#/definitions/StreamsKafkaSecurity"
+        },
+        "ClusterName": {
+          "type": "string",
+          "description": "Name of the cluster configured for this connection."
+        },
+        "DbRoleToExecute": {
+          "$ref": "#/definitions/DBRoleToExecute"
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "properties": {
+    "Profile": {
+      "type": "string",
+      "description": "The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).",
+      "default": "default"
+    },
+    "Name": {
+      "description": "Human-readable label that identifies the stream connection.",
+      "type": "string"
+    },
+    "DataProcessRegion": {
+      "$ref": "#/definitions/StreamsDataProcessRegion"
+    },
+    "StreamConfig": {
+      "$ref": "#/definitions/StreamConfig"
+    },
+    "Id": {
+      "description": "Unique 24-hexadecimal character string that identifies the project.",
+      "type": "string"
+    },
+    "Connections": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/StreamsConnection"
+      },
+      "insertionOrder": false
+    },
+    "GroupId": {
```

Comment:
For all our CFN and terraform resources we have been using project id over group id, I believe group id is an outdated term that remains only in the API side.